test/build_payment.js

```diff
@@ -7,6 +7,7 @@ describe('Ripple REST Client buildPayment', function() {
 
   before(function () {
     client = new Client({
+      api: 'http://localhost:5990/',
```

Comment:
ah, removed http://localhost:5990/ so it tests against api.ripple.com.
updated. thanks for catching it @wltsmrz

On Wed, Mar 11, 2015 at 2:42 PM, wltsmrz <notifications@github.com> wrote:

> In test/build_payment.js
> <https://github.com/ripple/ripple-rest-client/pull/1#discussion_r26258904>
> :
>
> > @@ -7,6 +7,7 @@ describe('Ripple REST Client buildPayment', function() {
> >
> >    before(function () {
> >      client = new Client({
> > +      api: 'http://localhost:5990/',
>
> What did you fix?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ripple/ripple-rest-client/pull/1/files#r26258904>.
>



-- 
*Abiy Seifu |* Software Engineer
*Ripple Labs, Inc.*
abiy@ripple.com | www.ripple.com